pkgs/applications/accessibility/espeakup/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, meson, ninja, espeak-ng, fetchFromGitHub, pkg-config, cmake, alsaLib, systemd }:
+
+stdenv.mkDerivation rec {
+  pname = "espeakup";
+  version = "0.90";
+  src = fetchFromGitHub {
+    owner = "linux-speakup";
+    repo = "espeakup";
+    rev = "569e9f5b0edbf427c5b877912ac4f016efd6a492";
+    sha256 = "sha256:0lmjwafvfxy07zn18v3dzjwwpnid2xffgvy2dzlwkbns8gb60ds2";
+  };
+
+  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
+
+  nativeBuildInputs = [ meson ninja pkg-config cmake alsaLib systemd ];
```

Comment:
`alsaLib` and `systemd` should probably got to `buildInputs` (if they are runtime, rather than build-time dependencies).

Reply:
Looks like the dependencies are still in `nativeBuildInputs`.